skills/gen-tts/generate-tts.py

```
@@ -404,6 +408,57 @@ def pcm_to_wav(
+    """Apply an ffmpeg atempo pass in-place. No-op for speed == 1.0."""
+    if speed == 1.0:
+        return
+    tmp_path = wav_path + ".speed.tmp.wav"
+    cmd = [
+        "ffmpeg", "-y", "-nostdin", "-loglevel", "error",
+        "-i", wav_path,
+        "-filter:a", atempo_filter_chain(speed),
+        tmp_path,
+    ]
+    try:
+        subprocess.run(cmd, check=True)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)